docs/css/style.css

```
@@ -188,6 +188,18 @@ img.screenshot {
   max-width: 100%;
 }
 
+#matomoLogo {
+    display: block; 
+    width: 176px; 
+    height: 176px;
+}
+
+.sponsorship {
+    display: flex; 
+    justify-content: center;
+	border-bottom: 10px solid white;
```

Let's use a margin and adjust the size to create some visual rhythm. `60px` is the size of the bottom padding from `<main>`, so that's what we need for symmetry

```suggestion
	margin-bottom: 60px;
```